packages/notifier-seeder/src/models/purpose/purposeEventNotificationMessage.ts

```diff
@@ -0,0 +1,58 @@
+import { PurposeEventEnvelopeV2 } from "pagopa-interop-models";
+import { match } from "ts-pattern";
+import { v4 as uuidv4 } from "uuid";
+import { QueueMessage } from "../../queue-manager/queueMessage.js";
+import { PurposeEventNotification } from "./purposeEventNotification.js";
+
+export const eventV2TypeMapper = (
+  eventType: PurposeEventEnvelopeV2["type"]
+): string =>
+  match(eventType)
+    .with("PurposeAdded", "PurposeCloned", () => "purpose_created")
+    .with("DraftPurposeUpdated", () => "purpose_updated")
+    .with(
+      "PurposeWaitingForApproval",
+      "NewPurposeVersionWaitingForApproval",
+      "PurposeVersionOverQuotaUnsuspended",
+      () => "purpose_version_waited_for_approval"
+    )
+    .with(
+      "PurposeActivated",
+      "NewPurposeVersionActivated",
+      "PurposeVersionActivated",
+      () => "purpose_version_activated"
+    )
+    .with(
+      "DraftPurposeDeleted",
+      "WaitingForApprovalPurposeDeleted",
+      () => "purpose_deleted"
+    )
+    .with(
+      "PurposeVersionUnsuspendedByProducer",
+      "PurposeVersionUnsuspendedByConsumer",
+      () => "purpose_version_activated"
```

Comment:
These could be grouped with the previous ones since they send the same kind `purpose_version_activated`

Reply:
Sure